### PR TITLE
Adding tags support for RunPowerShellOnTargetMachines

### PR DESCRIPTION
**Pull request by pryenugu**
- Adding tags support.

### CODE REVIEW

File: Tasks/PowerShellOnTargetMachines/task.json [hunk 1]
```diff
@@ -41,6 +41,14 @@
             "helpMarkDown": "Provide a comma separated list of machines for the deployment like Machine1, Machine2. Default is all machines in the Machine Group."
         },
         {
+            "name": "Tags",
```
Comment (kmkumaran) [inline Tasks/PowerShellOnTargetMachines/task.json]: I thought the UI should have a checkbox to ask customers if the filter criteria should be based on machine or Tags. The machine or tags should have their visibility set according to the checkbox.

File: Tasks/PowerShellOnTargetMachines/task.json [hunk 2]
```diff
@@ -41,6 +41,14 @@
...
+            "type": "string",
+            "label": "Deploy to Tags",
+            "defaultValue": "",
+            "required": false,
+            "helpMarkDown": "Provide a comma separated list of machines for the deployment like Tag1:Value1;Tag2:FirstValue,SecondValue. Default is all machines in the Machine Group."
+        },
```
Comment (kmkumaran) [inline Tasks/PowerShellOnTargetMachines/task.json]: 'comma' separated is not true. 'Semi-colon' separated.

Comment (pryenugu) [inline Tasks/PowerShellOnTargetMachines/task.json]: Resolved.

**Comment by kmkumaran** [comment]
Looks fine.

**Pull request merged**